src/java.base/share/classes/jdk/internal/reflect/MethodAccessorGenerator.java

```diff
@@ -92,7 +92,7 @@ public ConstructorAccessor generateConstructor(Class<?> declaringClass,
     {
         boolean isStaticFactory = declaringClass.isValue();
         return (ConstructorAccessor) generate(declaringClass,
-                                              "<init>",
+                                              isStaticFactory ? "<vnew>" : "<init>",
```

Comment:
Core reflection does not use this method accessor generator by default.   I don't expect this change is needed.  Are you seeing any test failure without this change and other relevant change in jdk.internal.reflect?

Reply:
It's gone, JDK-8281295 removed its use